libs/langgraph/src/pregel/index.ts

```diff
@@ -982,7 +1270,25 @@ export class Pregel<
     return patchCheckpointMap(nextConfig, saved ? saved.metadata : undefined);
   }
 
-  _defaults(config: PregelOptions<Nn, Cc>): [
+  /**
+   * Gets the default values for various graph configuration options.
+   * This is an internal method used to process and normalize configuration options.
+   *
+   * @param config - The input configuration options
+   * @returns A tuple containing normalized values for:
+   * - debug mode
+   * - stream modes
+   * - input keys
+   * - output keys
+   * - remaining config
+   * - interrupt before nodes
+   * - interrupt after nodes
+   * - checkpointer
+   * - store
+   * - whether stream mode is single
+   * @internal
+   */
+  _defaults(config: PregelOptions<Nodes, Channels>): [
```

Comment:
Not sure why we're documenting this (also not sure why this even exists in JS, in py it exists because we need to share it between sync and async methods)